### PR TITLE
Add test resource

### PR DESCRIPTION
**Pull request by GMartinez-Sisti**
Please proof the quick_start.md carefully as a lot was changed in there.

### CODE REVIEW

File: proxy/fyde_proxy_troubleshoot.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Troubleshoot
 
+- The steps in this page are best performed from a Desktop client
+
+- Since most Mobile clients don't provide the required tools, we are developing features to help troubleshooting from this type of platforms
```
Comment (oNaiPs) [inline proxy/fyde_proxy_troubleshoot.md]: @GMartinez-Sisti perhaps `Troubleshooting using the mobile app is currently in development.`?

File: quick_start.md [hunk 2]
```diff
@@ -131,9 +129,19 @@ Click **Create**.
...
 ![Add User](console/configurations/imgs/control_add_user.png)
 
-## Access created resource
+## Enroll a device
 
 - Enroll a new device for the created user by acessing the enrollment link
   - The link is sent to the email configured for the user or by sharing the link from the user details
 
-- Access the configured resource
+- The enrollment process needs to be performed from the device to be enrolled
```
Comment (oNaiPs) [inline quick_start.md]: @GMartinez-Sisti the 2x enrollment seems a bit confusing.
How about something like
```
The enrollment process is done directly from the created user's device.
```
Do we have any steps for this in this repo? If so, we could add a link to them.

File: quick_start.md [hunk 3]
```diff
@@ -1,10 +1,48 @@
...
+
+1. [Troubleshoot](#troubleshoot)
+
+    - If the device is not accessing the resource we provide these troubleshooting steps
+
+## Add new Fyde Access Proxy
+
+- To add a Fyde Access Proxy, go to the **Access tab** and navigate to the **Access Proxies**. Click the **“+”** icon in the top right.
+
+- This proxy will be used to test an access for the test resource, however this doesn't prevent it from being used as a permanent proxy later
```
Comment (oNaiPs) [inline quick_start.md]: later on.

File: proxy/fyde_proxy_troubleshoot.md (hunk 1 above)
Comment (GMartinez-Sisti) [reply proxy/fyde_proxy_troubleshoot.md]: That's better. Going to change.

File: quick_start.md (hunk 2 above)
Comment (GMartinez-Sisti) [reply quick_start.md]: Looking at it again, it looks confusing indeed. Changing.

File: quick_start.md (hunk 3 above)
Comment (GMartinez-Sisti) [reply quick_start.md]: Got it!

**Comment by GMartinez-Sisti** [comment]
@oNaiPs Suggestions pushed.

**Pull request merged**